src/components/core/NavLink/NavLink.tsx

```diff
@@ -22,11 +22,11 @@ export default function NavLink({ to, href, children, dataTestId, className, onC
     track('Link Click', {
       to: to || href,
     });
-  }, [href, to, track]);
 
-  if (onClick) {
-    onClick();
-  }
+    if (onClick) {
+      onClick();
+    }
```

Comment:
This seemed like a bug, so I fixed it. Didn't have any usages of `<NavLink onClick={...}>` so it never came up.

Reply:
awesome catch